lua/lspinstall/servers/elixir.lua

```diff
@@ -0,0 +1,11 @@
+local config = require'lspconfig'.elixirls.document_config
+require'lspconfig/configs'.elixirls = nil -- important, immediately unset the loaded config again
+
```

Comment:
Just add `config.default_config.cmd = { "./language_server.sh" }`

Reply:
I was getting some strange errors with the `:LspInstall` command completion before when tried with that line, but now it seems to be working

Reply:
Perhaps you were missing the `{` `}` before?

Reply:
I tried the first time just indexing the `cmd` field and setting a string value, but it didn't existed, so I also tried to create it before doing that, (which is the same as the line you sent) and both don't cause errors but the lsp doesn't seem to start.
It has started before because I've forgot that I had hardcoded the path in my vimrc's elixirls setup for testing.

Reply:
what does `:LspInfo` say?

Reply:
```
Configured servers: elixirls
Neovim logs at: /home/<user>/.cache/nvim/lsp.log

0 client(s) attached to this buffer: 

0 active client(s): 

Clients that match the filetype elixir:
  
  Config: elixirls
  	cmd:               cmd not defined
  	cmd is executable: cmd not defined
  	identified root:   /home/<user>
  	custom handlers:   
```

Reply:
I'll commit the current code

Reply:
Needs to be `./elixir-ls/language_server.sh`. The `./` is important to be recognized as a local path.

Reply:
Oh yes, I forgot the `./`, but even adding it the behaviour and `:LspInfo` output are still the same.

Reply:
That's very weird. I suspect something in your config is wrong because the code looks fine. Also I just tested it, and it works on my machine.